app/experimenter/nimbus-ui/src/components/TableMetricPrimary/index.tsx

```diff
@@ -3,9 +3,88 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
+import { AnalysisData } from "../../lib/visualization/types";
+import TableVisualizationRow from "../TableVisualizationRow";
+import { getExperiment_experimentBySlug_primaryProbeSets } from "../../types/getExperiment";
+import {
+  PRIMARY_METRIC_COLUMNS,
+  TABLE_LABEL,
+  DISPLAY_TYPE,
+} from "../../lib/visualization/constants";
 
-const TableMetricPrimary: React.FunctionComponent = () => {
-  return <div data-testid="table-metric-primary">primary metric table</div>;
+type PrimaryMetricStatistic = {
+  name: string;
+  displayType: DISPLAY_TYPE;
+  branchComparison?: string;
+  value?: string;
+};
+
+type TableMetricPrimaryProps = {
+  results: AnalysisData["overall"];
+  probeSet: getExperiment_experimentBySlug_primaryProbeSets | null;
+};
+
+const getStatistics = (
+  probeset: string | null,
+): Array<PrimaryMetricStatistic> => {
+  const probesetMetricID = `${probeset}_ever_used`;
+
+  // Make a copy of `PRIMARY_METRIC_COLUMNS` since we modify it.
+  const primaryMetricStatisticsList = PRIMARY_METRIC_COLUMNS.map(
+    (statistic: PrimaryMetricStatistic) => {
+      statistic["value"] = probesetMetricID;
+      return statistic;
+    },
+  );
+
+  return primaryMetricStatisticsList;
+};
+
+const TableMetricPrimary = ({
+  results = {},
+  probeSet,
+}: TableMetricPrimaryProps) => {
+  const primaryMetricStatistics = getStatistics(probeSet!.slug);
+  const metricKey = `${probeSet?.slug}_ever_used`;
+
+  return (
+    <div data-testid="table-metric-primary">
+      <h2 className="h5 my-3">{probeSet?.name}</h2>
+      <table className="table text-right mb-5 mt-4">
```

Comment:
What do you think about centering this table?

`text-right`:
![image](https://user-images.githubusercontent.com/13018240/99691905-544c4680-2a4f-11eb-8fd7-76f2da611b0d.png)

`text-center`:
![image](https://user-images.githubusercontent.com/13018240/99691967-629a6280-2a4f-11eb-9462-1274e33925fa.png)


Fine by me if you preferred right-aligned, just wanted to throw it out there since I thought the Results table looked better centered.

Reply:
oh. I had centered it and forgot to push it! hah